ibis/formats/tests/test_pyarrow.py

```diff
@@ -84,6 +81,10 @@ def test_non_roundtripable_types(arrow_type, ibis_type, restored_type):
     assert_dtype_roundtrip(arrow_type, ibis_type, restored_type)
 
 
+def test_timestamp_with_no_scale():
+    assert dt.Timestamp.from_unit("us").to_pyarrow() == pa.timestamp("us")
```

Comment:
I guess this should be 'dt.Timestamp(scale=None)'

Reply:
Yup!